fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,236 @@
+
+We have a sample package at https://github.com/GoogleCloudPlatform/anthos-config-management-samples/tree/main/fleet-packages-quickstart/config. This is a simple example that just sets up nginx. Fork this repo into your own GitHub account.
+
+Fleet Packages leverages [Cloud Build Repositories] (2nd gen) for connecting to git. Follow these steps to set it up:
+
+1. Enable the Cloud Build and Secret Manager APIs:
+```
+gcloud services enable cloudbuild.googleapis.com
+gcloud services enable secretmanager.googleapis.com
+```
+
+2. Verify the IAM permissions as described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#iam_perms.
+
+3. Create a connection to your GitHub host using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_host. You can use either the console or gcloud. You must use the region `us-central1` and we suggest using the name `fp-quickstart` for the connection (the rest of the guide will use this name).
+
+4. Connect to the forked GitHub repository using the steps described in https://cloud.google.com/build/docs/automating-builds/github/connect-repo-github?generation=2nd-gen#connecting_a_github_repository_2. You can use either the console or gcloud. Again, you must use the region `us-central1` and we suggest using the name `anthos-config-management-samples` for the repository.
+
+5. We will need the fully qualified name for the repository a little later. With the suggested names, it will be:
+
+```
+projects/$PROJECT_ID/locations/us-central1/connections/fp-quickstart/repositories/anthos-config-management-samples
+```
+
+If you are uncertain about then names, you can list these with gcloud using the following commands:
+
+List all connections:
+
+```
+gcloud builds connections list --region=us-central1 --project=$PROJECT_ID
+```
+
+List all repositories for a specific connection:
+
+```
+gcloud builds repositories list --region=us-central1 --connection=<CONNECTION_NAME>
+```
+
+## Configure a Service Account
+
+Create a service account:
+
+```
+gcloud iam service-accounts create fp-quickstart-sa
+```
+
+Add an IAM policy binding for the ResourceBundle Publisher role:
+
+```
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+   --member="serviceAccount:fp-quickstart-sa@$PROJECT_ID.iam.gserviceaccount.com" \
+   --role='roles/configdelivery.resourceBundlePublisher' \
+   --condition=None
+```
+
+Add an IAM policy binding for the [Logs Writer] role:
+
+```
+gcloud projects add-iam-policy-binding $PROJECT_ID \
+   --member="serviceAccount:fp-quickstart-sa@$PROJECT_ID.iam.gserviceaccount.com" \
+   --role='roles/logging.logWriter' \
+   --condition=None
+```
+
+## Verify that clusters are ready
+
+To verify that the clusters are ready, check that they are marked at `RUNNING` when running this command:
+
+```
+gcloud container clusters list --project=$PROJECT_ID
```

Comment:
this might be in line with how the rest of the guides here work but sometimes it's nice to include a sample output.

Reply:
Yeah, I agree. I plan to do another pass on this and add some more examples of output, both in the cli and in the UI.